### ISSUE TITLE
Extension does not work with process template inherited from out-of-the-box Agile process template

### ISSUE THREAD

**Issue by billcharlton0028** [jpiquot/ChildTasksTemplate#3]
I installed version 1.0.6 of your extension on Dec 19, 2020 at 8 pm EST on Azure DevOps Server 2019, Update 1.1, patch 6

Here is my json file:

```
{
  "tasks": [
    {
      "name": "Design Task",
      "fields": [
        {
          "name": "System.Title",
          "value": "{System.Title} Design FOO"
        }
      ]
    },
    {
      "name": "Dev Task",
      "fields": [
        {
          "name": "System.Title",
          "value": "{System.Title} Dev FOO"
        }
      ]
    },
    {
      "name": "Test Task",
      "fields": [
        {
          "name": "System.Title",
          "value": "{System.Title} Test FOO"
        }
      ]
    }
  ]
}
```

For a project based on the out-of-the-box Microsoft Agile process template, the template works great

For a project based on a custom process template that I inherited from the out-of-the-box Microsoft Agile process template, it does not work. Note: I created the customized process template using inheritance from the web portal and NOT by exporting/editing XML. Everything works as its supposed to except: When I'm editing a user story and click the `Add tasks` button, nothing happens, and the Edge console shows the following (see screen shot). 

If you want, I'll capture logs and send them to you. Let me know
 
![2020_12_20_12_48_13_User_Story_2786_new_test_Boards_and_1_more_page_Work_Microsoft_Edge](https://user-images.githubusercontent.com/28391637/102720691-c967a180-42c3-11eb-97fd-8b1a1f127d46.png)

**Comment by jpiquot** [comment]
Hi,

I can reproduce the error now. It seems that the issue comes from Kanban columns fields. I added a try/catch on this error to log on console instead of throwing an uncatched exception.

> Created task 10911 ChildTasksService.ts:101:16
> An error accured while setting field value. Name 'WEF_87E142FB9098434680E3C40F48B1A790_Kanban.Column.Done'; Value 'false'.can't assign to property "Done" on "Waiting user response": not an object ChildTasksService.ts:123:16
> An error accured while setting field value. Name 'WEF_259A92C5D3BE40C28675E8E6A87AA6AD_Kanban.Column.Done'; Value 'false'.can't assign to property "Done" on "Active": not an object ChildTasksService.ts:123:16
> An error accured while setting field value. Name 'WEF_87E142FB9098434680E3C40F48B1A790_Kanban.Column.Done'; Value 'false'.can't assign to property "Done" on "Waiting user response": not an object ChildTasksService.ts:123:16
> An error accured while setting field value. Name 'WEF_259A92C5D3BE40C28675E8E6A87AA6AD_Kanban.Column.Done'; Value 'false'.can't assign to property "Done" on "Active": not an object ChildTasksService.ts:123:16
> An error accured while setting field value. Name 'WEF_87E142FB9098434680E3C40F48B1A790_Kanban.Column.Done'; Value 'false'.can't assign to property "Done" on "Waiting user response": not an object ChildTasksService.ts:123:16
> An error accured while setting field value. Name 'WEF_259A92C5D3BE40C28675E8E6A87AA6AD_Kanban.Column.Done'; Value 'false'.can't assign to property "Done" on "Active": not an object ChildTasksService.ts:123:16
> An error accured while setting field value. Name 'WEF_87E142FB9098434680E3C40F48B1A790_Kanban.Column.Done'; Value 'false'.can't assign to property "Done" on "Waiting user response": not an object ChildTasksService.ts:123:16
> An error accured while setting field value. Name 'WEF_259A92C5D3BE40C28675E8E6A87AA6AD_Kanban.Column.Done'; Value 'false'.can't assign to property "Done" on "Active": not an object ChildTasksService.ts:123:16
> Created task 10912
>

**Comment by jpiquot** [comment]
Hi, this has been fixed in version 1.0.7 (april 11)
